### DISCUSSION TITLE
500: Internal Server Error - Advice needed.

### DISCUSSION THREAD

**Discussion by mrtargaryen** [wallabag/wallabag#7123] [wallabag.it] [answered]
```
500: Internal Server Error
Connection could not be established with host "ssl://127.0.0.1:465": stream_socket_client(): Unable to connect to ssl://127.0.0.1:465 (Connection refused)
```

I just installed Wallabag on my Windows 11 PC, and I used the following:

`$ docker run --name wallabag --link wallabag-db:wallabag-db -e "MYSQL_ROOT_PASSWORD=PASSWORD" -e "SYMFONY__ENV__DATABASE_DRIVER=pdo_mysql" -e "SYMFONY__ENV__DATABASE_HOST=wallabag-db" -e "SYMFONY__ENV__DATABASE_PORT=3306" -e "SYMFONY__ENV__DATABASE_NAME=wallabag" -e "SYMFONY__ENV__DATABASE_USER=wallabag" -e "SYMFONY__ENV__DATABASE_PASSWORD=wallapass" -e "SYMFONY__ENV__DATABASE_CHARSET=utf8mb4" -e "SYMFONY__ENV__DOMAIN_NAME=http://localhost" -p 80:80 wallabag/wallabag`

I'm a little in experienced and just wondering what the procedure for resolving the error is? Any help is really appreciated.  The error occurs when trying to register a new user.

(also, I'm not looking to use Wallabag externally from the Web, I just want to use it locally.)

**Comment by mrtargaryen** [comment]
I managed to log in with UN: wallabag PW: wallabag 

After I installed the Chrome extension on Brave browser and tested the save function, I noticed something wrong with the text, please see this image:

https://i.imgur.com/yeRDp1E.png

What would cause this? Any help setting Wallabag up would be appreciated, so far it's been the most frustrating container I've setup out of about 20, I just keep being confronted with problems.

**Reply by HolgerAusB** [reply]
did you just test it, with the github page or with others too? We need the article link to check if there is a need for a site-depended config, while github pages are generally a problem because of massive use of JavaScript.

**Reply by mrtargaryen** [reply]
Thanks for the reply, really appreciate the help.

Here's another screenshot (had to crop it, imgur won't allow the full shot for some reason)

https://i.imgur.com/bi7fZ1H.png

If you notice the words on wallabag they're all scrambled for some reason.

Here are the links I tested:

https://edition.cnn.com/middleeast/live-news/israel-hamas-war-gaza-news-12-18-23/index.html

https://www.abc.net.au/news/2023-12-18/qld-record-flooding-far-north-monday/103239260

Reddit appears to be blocked.

Edit1:

When I access the local IP of the PC Wallabag is on from my Mac Mini I see this (please follow the link):

https://i.imgur.com/fHql6wY.png

I have installed and used 20 other Containers in Docker and I haven't had this much trouble from any of those, I just don't know what these problems could be,

**Reply by mrtargaryen** [reply]
Okay, it appears to be a problem with Brave browser, as I installed Wallabag on Docker on my Mac Mini and the same issue appears when using Brave. But when I use Safari the problem doesn't occur. I also just tested Wallabag on Microsofts browser Edge and the same problem is present.

**Reply by HolgerAusB** [reply, marked as answer]
Now I could see the problem, but I am not a dev or docker pro. Instead of text, there should be icons from materials.css, like https://imgur.com/4CMov34

 Maybe you should search for "icons" in the issue section. Here are some, I found. Maybe they could help:
https://github.com/wallabag/wallabag/issues/5807
https://github.com/wallabag/docker/issues/146
https://github.com/wallabag/wallabag/issues/4993

in short, the docker run or compose.yml should contain the same domain or ip, as you use with your browser - I think.

And yes, reddit uses JavaScript as well. No fetch with wallabag UI. Maybe it works with 'wallabagger' browser plugin and activated browser content

**Reply by mrtargaryen** [reply]
In the Docker run command, I had put http://localhost, and not http://192.168.50.125 ... that was the problem. Your suggestion was correct and solved the issue. Thank you very much, appreciated!!
